cert/test_helpers.go

```diff
@@ -1,28 +1,50 @@
-package e2e
+package cert
+
+//TODO: this should only be built if performing tests
```

Comment:
I really dont want to require we use a tag to execute tests but it also feels bad to include this in the binary and I don't know of another way to deal with this without just copying these functions all over the place again.

Reply:
can't you define these in a `something_helpers_test.go` and use them in other `*_test.go` files?

Reply:
I desperately want there to be a way but the only thing I can come up with is a build tag much like how `e2e` works